x-pack/plugins/maps/public/content_management/duplicate_title_check.ts

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Why are we moving away from a common title duplicate check? Would it be worthwhile to move this to contentManagement plugin and pass in generic `searchTitles` props?

Reply:
We could be building the new `DuplicateTitleChekc` component but that was out of scope for this PR and it is not currently in planned. We could think of a generic solution / component for contents to use but we don't have it yet.